### PR TITLE
Update brew quietly

### PR DESCRIPTION
**Pull request by amadeu01**
Update quietly if VerboseLog is set to false.

### CODE REVIEW

**Comment by Roland-Bak** [comment]
Hey there @amadeu01!

Can you let us know what the use case for this would be?  The step already has an input to pass custom options, so the -quiet flag can be passed right now.

**Comment by adborbas** [comment]
Hi @amadeu01!

Actually custom options cannot be passed to the `update` command. But we would still be interested in the use-case for passing the `-quite` flag scenario. Thanks!

File: main.go [hunk 1]
```diff
@@ -90,7 +90,12 @@ func main() {
 	log.SetEnableDebugLog(cfg.VerboseLog)
 
 	log.Infof("Update homebrew")
-	cmd := command.New("brew", "update").SetStdout(os.Stdout).SetStderr(os.Stderr)
+	if cfg.VerboseLog {
+		cmd := command.New("brew", "update").SetStdout(os.Stdout).SetStderr(os.Stderr)
```
Comment (adborbas) [inline main.go:94]: Please define `cmd` outside the if-else scope, so it will be visible.

Comment (amadeu01) [reply main.go:94]: Okay! Thanks! I'll update!

**Pull request closed without merging**